File: 1. Beginner/1006/1006.hs
import Text.Printf

main :: IO ()
main = do
    a <- readLn :: IO Double
    b <- readLn :: IO Double
    c <- readLn :: IO Double
    let r = (a * 2 + b * 3 + c * 5) / 10
    printf "MEDIA = %.1f\n" r

File: 1. Beginner/1006/1006.ml
Scanf.scanf "%f\n%f\n%f" (fun a b c ->
    Printf.printf "MEDIA = %.1f\n" (((a) *. 2.0 +. (b) *. 3.0 +. (c) *. 5.0) /. 10.0);
)